AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/RemoteSyncEngine+IncomingEventReconciliationQueueEvent.swift

```diff
@@ -48,6 +48,8 @@ extension RemoteSyncEngine {
             remoteSyncTopicPublisher.send(.subscriptionsPaused)
         case .mutationEvent(let mutationEvent):
             remoteSyncTopicPublisher.send(.mutationEvent(mutationEvent))
+        case .mutationEventDropped:
```

Comment:
`mutationEventDropped` is no use to customer, can break here without sending back to `remoteSyncTopic`